src/Energy/Battery/ChargingHistory_v1.0.py

```diff
@@ -74,15 +74,16 @@ class ChargingHistoryResponse(CamelCaseModel):
     total_count: int = Field(
```

Comment:
Should the above field also have a description? (And sorry, can't get a comment added to the right line, because it wasn't changed).